spec/jara/releaser_spec.rb

```diff
@@ -483,16 +489,31 @@ module Jara
         s3_puts.last[:content_md5].should == '07BzhNET7exJ6qYjitX/AA=='
       end
 
-      it 'sets metadata that includes who built the artifact, the full SHA, the Git remote' do
+      it 'sets metadata that includes who built the artifact' do
         production_releaser.release
         s3_puts.last[:metadata]['packaged_by'].should include(%x(whoami).strip)
         s3_puts.last[:metadata]['packaged_by'].should match(/^.+@.+$/)
+      end
+
+      it 'sets legacy metadata with the full SHA and the Git remote' do
+        production_releaser.release
         s3_puts.last[:metadata].should include(
           'sha' => sha,
           'remote' => 'git@example.com:foo/bar',
         )
       end
 
+      it 'sets Git metadata with the full SHA, the Git remote, author and title' do
+        production_releaser.release
+        s3_puts.last[:metadata].should include(
+          'git_sha' => sha,
+          'git_remote' => 'git@example.com:foo/bar',
+          'git_author_name' => 'Some Author',
+          'git_author_email' => 'email@example.com',
+          'git_title' => 'This is a commit doing something',
+        )
+      end
```

Comment:
Now when there are multiple tests that describe the metadata, perhaps you could reformat this section like this:

```ruby
context 'sets metadata that' do
  it 'includes who built the artifact'
  it 'includes legacy details like SHA and Git remote'
  it 'includes Git details like the full SHA, remote, commit author and title'
end
```